packages/ui-components/src/data-display/ProductTile/ProductTile.tsx

```diff
@@ -1,41 +1,58 @@
 import React, { FC } from 'react';
 import { Card } from '../Card/'
+import { Product } from '@team21/types'
+import { useNavigate, } from 'react-router-dom'
+import "./index.css"
 
 type Props = {
-    product: any; // TODO
+    product: Product;
 };
 
 type BodyProps = {
     description: string;
     price: number;
-    qty: number;
+    quantity: number;
 }
 
-const ProductTileBody: FC<BodyProps> = ({description, price, qty}) => ( 
-    <> 
-        <div className="my-8">
-            <p className="text-base text-darkGray"> {description} </p>
+const ProductTileBody: FC<BodyProps> = ({description, price, quantity}) => { 
+    function setDescription(description:string){
+        if(description.length < 90)
+            return description
+        return `${description.slice(0, 90)}...` 
+    }
+    return <React.Fragment> 
+        <div className="description my-8">
+            <p className="text-base text-darkGray px-8 whitespace-normal break-all text-left"> {description && setDescription(description)} </p>
         </div>
         <div className="my-12">
             {price && <span className="text-xl text-dark font-semibold"> ${price} </span>}
         </div>
         <div className="my-8">
-            <span className={`${qty > 0 ? 'text-success' : 'text-red'} text-md font-semibold`}> {qty > 0 ? `In Stock: ${qty}` : 'Out of Stock'} </span>
+            <span className={`${quantity > 0 ? 'text-success' : 'text-red'} text-md font-semibold`}> {quantity > 0 ? `In Stock: ${quantity}` : 'Out of Stock'} </span>
         </div>
-    </>
-);
+    </React.Fragment>
+};
+
+const ProductTile: FC<Props> = ({ product }) => {
```

Comment:
you can use destructuring to avoid having to write product. everywhere
{ {name,image, .... } : product } something like this i think xD

Reply:
Yeah, ty for the advice.